public/assets/css/author.css

```diff
@@ -0,0 +1,242 @@
+aside {
+    width: 25%;
+    padding-left: 15px;
+    font-family: Anisse;
+}
+
+aside>header div {
+    display: inline-block;
+    vertical-align: middle;
+    padding: 15px;
+    width: 130px;
+}
+
+aside>header img{
+    width: 25%;
+    height: 25%;
+    border-radius: 50%;
+    border: 5px solid rgb(0, 162, 255);
+    filter: drop-shadow(0 0 0.35rem rgb(61, 58, 58));
+    cursor: pointer;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+aside>header img:hover {
+    filter: drop-shadow(0 0 0.35rem rgb(231, 227, 227));
+    box-shadow: 0 0 0.75rem rgb(0, 162, 255);
+}
+
+aside h1{
+    font-size: 12pt;
+    font-weight: 700; 
+}
+aside h2{
+    font-size: 9pt;
+}
+
+aside p {
+    font-size: 10pt;
+    padding-top: 30px;
+    padding-bottom: 20px;
+    text-align: justify;
+}
+
+.icon-rss {
```

Comment:
create a file path like this `css/icons/rss.css` and  `css/icons/icons.css`

in `css/icons/icons.css` would contain a global rule of the articles and would import the other files, like this :

in file `css/icons/icons.css`
```css
@import url(./rss.css)

.icon{
...
}
.icon:hover{
...
}
.icon-circle {
...
}
.icon-circle:hover{
...
}
```

in file `rss.css`
```css
.icon .rss{
...
}

.rss-xxx{
...
}

/* override color for icon-circle */
.icon-circle .rss-xxx{
...
}
```
or 
```css
.icon .icon-rss{
...
}

.icon-rss-xxx{
...
}

/* override color for icon-circle */
.icon-circle .icon-rss-xxx{
...
}
```

Reply:
"xxx" represent a generic name, like this exemple : "part-one", ...

```css
/* bem */
.icon-rss-part--one{
...
}

```
or 
```css
/* oocss */
.icon-rss-part-one{
...
}

```

Reply:
.css stylesheet created for each icon + stylesheet created for global rules